html-css/week1/Ali-Nematollahi/index.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Ali-Nematollahi-CV</title>
+    <link rel="stylesheet"
+        href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@24,400,0,0&icon_names=calendar_month,language,location_on,menu,schedule" />
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+
+    <div id="container">
+        <nav>
+            <ul>
+                <li>
+                    <a href="#">SUMMARY</a>
+                </li>
+                <li>
+                    <a href="#">SKILLS</a>
+                </li>
+                <li>
+                    <a href="#">PORTFOLIO</a>
+                </li>
+                <li>
+                    <a href="#">CONTACT</a>
```

Comment:
You've forgotten to add the links that would lead to each section
```suggestion
                    <a href="#summary">SUMMARY</a>
                </li>
                <li>
                    <a href="#skills">SKILLS</a>
                </li>
                <li>
                    <a href="#portfolio">PORTFOLIO</a>
                </li>
                <li>
                    <a href="#contact">CONTACT</a>
```